cookbook/profiler/storage.rst

```diff
@@ -4,7 +4,7 @@
 Switching the Profiler Storage
 ==============================
 
-By default the profile stores the collected data in files in the cache directory.
+By default the profile stores the collected data in files in the cache directory, under ``%kernel.cache_dir%/profiler``.
```

Comment:
:+1: for @javiereguiluz' proposal.

@jls2933 can you please edit your PR to use this rewording? If you don't have the time or you don't know how to do it, feel free to ask for help or we can do it after merging the PR.